tutorials/jmsActiveMQ/src/test/java/com/example/jmsActiveMQ/ApplicationTest.java

```diff
@@ -65,10 +66,14 @@ public void testApplication() throws IOException, Exception {
       File file = new File(FILE_PATH);
       file.delete();
       
+      getConfig();
+
+      qNameToUse = conf.get("dt.operator.amqIn.prop.subject");
```

Comment:
Minor point: Both qNameToUse and the brokerURL could be retrieved in getConfig() and used whereever they are needed.